interface/portraits/portraits_highelf.gfx

```diff
@@ -385,4 +385,390 @@ spriteTypes = {
 
 		headgear_that_hides_hair = { 9 }
 	}
+
+	### highelffel MALE ####
+
+	# CULTURE_INDEX: 0 = clothes behind, 1 = headgear behind, 2 = clothes, 3 = headgear mid, 4 = clothes infront, 5 = headgear
+
+	# highelf Male
+	portraitType = {
+		name = "PORTRAIT_highelffelgfx_male"
+		effectFile = "gfx/FX/portrait.lua"
+		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
+			"GFX_character_background:p0"
+			"GFX_highelf_male_clothes_behind:p3"
+			"GFX_highelf_male_headgear_behind:p5"
+			"GFX_highelf_male_hair_behind:p1:h:y"
+			"GFX_highelf_male_beard_behind:p4:h:y"
+			":p5:c6"
+
+			"GFX_highelf_male_base:p2"
+			"GFX_highelf_male_neck:d0"
+			"GFX_highelf_male_clothes:p3"
+			"GFX_highelf_male_cheeks:d4"
+			"GFX_highelf_male_chin:d1"
+			"GFX_highelf_male_mouth:d2"
+			"GFX_highelf_male_nose:d3"
+			"GFX_highelf_male_eyes:d6"
+			"GFX_highelf_male_eyes2:d6:e"
+			"GFX_highelf_male_headgear_mid:p5"
+			"GFX_highelf_male_ear:d7"
+
+			"GFX_empty:p7:y"
+			"GFX_empty:p8"
+			"GFX_empty:p9"
+			"GFX_empty:p10"
+			"GFX_empty:p13:y:o32x63"
+			":p5:c7"
+
+			"GFX_highelf_male_hair:p1:h:y"
+			"GFX_highelf_male_beard:p4:h:y"
+			"GFX_empty:p12:y:o40x45"
+			"GFX_highelf_male_headgear:p5"
+			"GFX_highelf_male_clothes_infront:p3"
+			
+			"GFX_belf_lore_chars:p26"
+			"GFX_elf_druid_form:p19"
+			"GFX_character_imprisoned:p6"
+			"GFX_being_overlay:p20"
+			"GFX_player_overlay:p11"
+		}
+
+		hair_color_index = 8
+		hair_color = { # dark, base, highlight
+			#Standart colors
+			{ 0 0 0 } { 196 172 91 } { 255 255 255 } # Blonde
+			{ 0 0 0 } {  75 75 75 } { 255 255 255 } # Black
+			{ 0 0 0 } { 176 155 108 } { 255 255 255 } # Hay
+			{ 0 0 0 } { 125 85 56 } { 255 255 255 } # Brown
+			{ 0 0 0 } { 150 90 55 } { 255 255 255 } # Reddish brown
+			{ 0 0 0 } { 178 80 41 } { 255 255 255 } # Ginger
+			{ 0 0 0 } { 78 54 37 } { 255 255 255 } # Dark brown
+			
+			#Anime colors
+			{ 0 0 0 } { 125 125 175 } { 255 255 255 } # Dark Blue
+			{ 0 0 0 } { 235 235 255 } { 255 255 255 } # White Blue
+		}
+
+		eye_color_index = 9
+		eye_color = {
+			{ 124 252 000 } # former Bright Blue
+			{ 173 255 047 } # former Blue
+			{ 000 255 000 } # former Light Blue
+			{ 070 255 000 } # former Violet
+		}
```

Comment:
Can we use human eye colors? A lot of eye colors per every race is pretty confusing.
```suggestion
		eye_color = {
			{ 125 255 125 } #Green
			{ 200 255 30 } #Yellowish Green
		}
```